File: Samples_2_1_0/WebApplication/Features/SendToUser/SendToUserHub.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Microsoft.AspNet.SignalR;

namespace WebApplication.Features.SendToUser
{
    public class SendToUserHub : Hub
    {
        public void SendToMe(string value)
        {
            Clients.User(Context.User.Identity.Name).message(value);
        }

        public void SendToUser(string userId, string value)
        {
            Clients.User(userId).message(value);
        }

        public void SendToUsers(IList<string> userIds, string value)
        {
            Clients.Users(userIds).message(value);
        }
    }
}

File: Samples_2_1_0/WebApplication/Features/SendToUser/SendToUserHub.js
function writeError(line) {
    var messages = $("#messages");
    messages.append("<li style='color:red;'>" + getTimeString() + ' ' + line + "</li>");
}

function writeLine(line) {
    var messages = $("#messages");
    messages.append("<li style='color:black;'>" + getTimeString() + ' ' + line + "</li>");
}

function getTimeString() {
    var currentTime = new Date();
    return currentTime.toTimeString();
}

$(function () {
    var connection = $.connection.hub,
        hub = $.connection.sendToUserHub,
        message = $("#message"),
        userId = $("#userId"),
        userIdsMessage = $("#userIdsMessage"),
        userIds = $("#userIds");

    connection.logging = true;

    hub.client.message = function (value) {
        writeLine(value);
    }

    connection.start()
        .done(function () {
            writeLine("connected");
        })
        .fail(function (error) {
            writeError(value);
        });

    $("#sendToMe").click(function () {
        hub.server.sendToMe(message.val());
    });

    $("#sendToUser").click(function () {
        hub.server.sendToUser(userId.val(), message.val());
    });

    $("#sendToUsers").click(function () {
        var listUsers = userIds.val().split(";");
        hub.server.sendToUsers(listUsers, userIdsMessage.val());
    });
});